packages/esm-billing-app/src/claims/dashboard/table/claims-dashboard-table.scss

```
@@ -0,0 +1,181 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@import '~@openmrs/esm-styleguide/src/vars';
```

```suggestion
@use '@carbon/color'
```
Seems moving forward we want to retire this import